.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ executors:
 parameters:
     current_golden_images_hash:
         type: string
-        default: cb80d9c57e1307d31e1d372cafe1da7f84332664
+        default: 41a01f58c286cd3c2c865fc22d4cae847670ccab
```

Comment:
Looks like you pointed this to the build from last week, rather than the one from earlier today: https://app.circleci.com/pipelines/github/adobe/spectrum-web-components/4458/workflows/9d5a70d2-f60f-45ea-bc29-b2c5fd6f1e72/jobs/27653